### ISSUE TITLE
No navigation bar screen button select sounds with `?homeScreen=false`

### ISSUE THREAD

**Issue by zepumph** [phetsims/joist#649]
While working on a bug in https://github.com/phetsims/joist/issues/641#issuecomment-668865380, I discovered this.

To reproduce:
1. Load a multiscreen sim with `homeScreen=false`
2. select screen buttons and there will be no sound.

@jbphet, is this a bug or feature?

**Comment by jbphet** [comment]
In https://github.com/phetsims/tambo/issues/91, the sound generator was moved to the `HomeScreenView` class because issues were coming up for single screen sims.  It appears that @zepumph has found a case where this doesn't work out well, since the home screen - and thus the sound generator - is never created in this test case.  @samreid hadn't anticipated this case at the time this sound generator was added.  I'll see what I can figure out.

**Comment by jbphet** [comment]
I fixed this by separating out the sound generation that occurs for the home screen and that which occurs when switching screens.  This seems to work, at least in the test cases that I could think of.  I was a little unclear about the distinction between some of the properties that I ended up using, so I logged #655.  I essentially used trial and error to figure out which changes would work to initiate which sounds, so it would be good for @samreid to have a look and make sure the approach I took is reasonable.
